public/app/features/plugins/admin/components/InstallControls/InstallControlsWarning.tsx

```diff
@@ -24,67 +24,94 @@ export const InstallControlsWarning = ({ plugin, pluginStatus, latestCompatibleV
   const isCompatible = Boolean(latestCompatibleVersion);
 
   if (plugin.type === PluginType.renderer) {
-    return <div className={styles.message}>Renderer plugins cannot be managed by the Plugin Catalog.</div>;
+    return (
+      <Alert
+        severity="warning"
+        title="Renderer plugins cannot be managed by the Plugin Catalog."
+        className={styles.alert}
+      />
+    );
   }
 
   if (plugin.type === PluginType.secretsmanager) {
-    return <div className={styles.message}>Secrets manager plugins cannot be managed by the Plugin Catalog.</div>;
+    return (
+      <Alert
+        severity="warning"
+        title="Secrets manager plugins cannot be managed by the Plugin Catalog."
+        className={styles.alert}
+      />
+    );
   }
 
   if (plugin.isEnterprise && !featureEnabled('enterprise.plugins')) {
     return (
-      <HorizontalGroup height="auto" align="center">
-        <span className={styles.message}>No valid Grafana Enterprise license detected.</span>
-        <LinkButton
-          href={`${getExternalManageLink(plugin.id)}?utm_source=grafana_catalog_learn_more`}
-          target="_blank"
-          rel="noopener noreferrer"
-          size="sm"
-          fill="text"
-          icon="external-link-alt"
-        >
-          Learn more
-        </LinkButton>
-      </HorizontalGroup>
+      <Alert
+        severity="warning"
+        title="This is a development build of the plugin and can&#39;t be uninstalled."
```

Comment:
Is this title correct? Looks like a duplicate of `isDev` condition below.

Reply:
Good catch! Thank you!